core-data/index.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * WordPress Dependencies
+ */
+import {
+	registerReducer,
+	registerSelectors,
+	registerResolvers,
+} from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import reducer from './reducer';
+import { getCategories } from './selectors';
+
+/**
+ * Module Constants
+ */
+const MODULE_KEY = 'core';
+
+const store = registerReducer( MODULE_KEY, reducer );
```

Comment:
Should we use the `createStore` API? Should resolvers be integrated into this API?

Reply:
True, it was not there yet when I started the PR ;)